### DISCUSSION TITLE
signUpEmailPassword function is deprecated?

### DISCUSSION THREAD

**Discussion by ankur1163** [nhost/nhost#1464] [Q&A]
In vs code, i see strike through on signupemailpassword. Is this deprecated or its something else?

https://prnt.sc/2CYzqW8P4G48

**Comment by elitan** [comment]
Looks like an issue in our SDK, I've created an issue here: https://github.com/nhost/nhost/issues/1465

**Reply by plmercereau** [reply]
`email` is probably of `unknown` type. Please try passing on a `string` email.

**Comment by ankur1163** [comment]
when this issue will be solved?
